fbgemm_gpu/include/fbgemm_gpu/hipcub_namespace_postfix.hpp

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Do we need this .hpp file ? (dup with .cuh file above)

Reply:
The same as above.